src/pke/decryption.cpp

```diff
@@ -7,12 +7,12 @@ using namespace lbcrypto;
 namespace py = pybind11;
 
 template<typename Element>
-Plaintext DecryptWrapper(Ciphertext<Element> ciphertext,const PrivateKey<Element> privateKey){
+Plaintext DecryptWrapper_standalone(Ciphertext<Element> ciphertext,const PrivateKey<Element> privateKey){
```

Comment:
I see that there is also a `DecryptWrapper`. Can you add a comment on why there are separate implementations? So that someone doesn't have to hunt across the files to determine the difference. 

In both locations, reference the other implementation and detail why this is different from that